sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -174,6 +174,7 @@ private void stopPreviousTransactions() {
     }
   }
 
+  // TODO should we start a new trace here if performance is disabled?
```

Comment:
Need help deciding whether this should create a new `PropagationContext` with random `traceId` and `spanId` or if we rather want to keep the previous trace going.